_includes/blog-articles.html

```diff
@@ -0,0 +1,32 @@
+<!-- Articles -->
+<div class="container">
+  <div class="row pt-3">
+  {% for blog in site.data.blogs[site.locale].articles %}
+	<div class="col-sm-4 pb-5" id="blogCard" >
+	<a href="{{blog.blog-page}}">
+	  <div class="card">
+		<div class="card__header">
+		  <img src="{{blog.image}}" alt="card__image" class="card__image" width="350">
```

Comment:
Any reason for the double underscore "__" throughout? Single underscore would be better coding convention.

Reply:
No real reason, probably an oversight when using publicly available code as     inspiration.